packages/medusa/src/api/routes/admin/sales-channels/get-sales-channels.ts

```diff
@@ -0,0 +1,29 @@
+import { Response } from "express"
```

Comment:
todo: Rename file. Should be singular: `get-sales-channel.ts`. Change should be reflected in `index.ts` as well